maven-help-plugin/src/main/java/org/apache/maven/plugins/help/EvaluateMojo.java

```diff
@@ -373,7 +373,7 @@ private void handleResponse( String expr )
 
         if ( obj != null && expr.equals( obj.toString() ) )
         {
-            getLog().warn( "The Maven expression was invalid. Please use a valid expression." );
+            System.out.println( "The Maven expression was invalid. Please use a valid expression." );
```

Comment:
Probably this and the previous one should use `System.err`, to avoid sending junk text to a program reading stdout.